index/src/registry/res/erc20-abi.json

```diff
@@ -0,0 +1,156 @@
+[
```

Comment:
Can we move this `abi.json` into the contracts/ directory?

Reply:
If `index` contains stuff to interact with EVM contract, it's ok to put it here. Why you said it should belong to the application layer?